Update README.md - new presentation (#23)
SAP/project-portal-for-innersource: README.md

A running instance of the portal using mock data can be found [here](https://sap.github.io/project-portal-for-innersource/index.html).

> The portal can also be seen in action at the InnerSource Commons Fall Summit 2020 session [The Unexpected Path of Applying InnerSource Patterns](https://www.youtube.com/watch?v=6r9QOw9dcQo), and, more recently, at the July 2021 InnerSource Commons Community Call [InnerSource & Discoverability](https://youtu.be/Yi2iVMa-gxM).

Reviewer: ```suggestion
The portal can also be seen in action in these recordings:
- [The Unexpected Path of Applying InnerSource Patterns](https://www.youtube.com/watch?v=6r9QOw9dcQo) - InnerSource Commons Fall Summit 2020 session
- [InnerSource & Discoverability](https://youtu.be/Yi2iVMa-gxM) - July 2021 InnerSource Commons Community Call
```

I think the readers care more about the links, than where the recordings happened.
Therefore I converted this into a list and pulled the links to the front.

This would also allow you to reorder the list easily (e.g. in case you want to mention the Community Call first), as well as adding further presentations in the future.
